evaluation/few_shot_eval.py

```diff
@@ -0,0 +1,699 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright BigScience, The HuggingFace Team and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Reproduce the main evaluation in `Multitask Prompted Training Enables Zero-Shot Task Generalization` using PyTorch.
+
+This script is heavily adapted from https://github.com/huggingface/transformers/blob/7533d30acd975027e83a548e4c38e06fa335291b/examples/pytorch/multiple-choice/run_swag_no_trainer.py
+"""
+
+import argparse
+import logging
+import os
+import random
+from dataclasses import dataclass
+from itertools import chain
+from typing import Optional, Union
+import csv
+import math
+
+import datasets
```

Comment:
hmm... they are already set as requirements... https://github.com/bigscience-workshop/promptsource/blob/4a662713df656d37ee1f3fb8ee7966cb2b8e5a74/requirements.txt#L8

Reply:
They are in the requirements, but the `requirements.txt` is not used in `setup.py`. I added a proposal to fix that https://github.com/bigscience-workshop/promptsource/pull/711

Reply:
oh yes you are totally right, sorry for some reason i was sure the requirements were called in the setup.py file but i was wrong.

i will accept your PR. note @Guitaricet  that since the requirements points to promptsource v0.1 (the version we used for t zero), this PR won't be reflected, so let's just add this to the requirements here